can_tools/scrapers/official/base.py

```diff
@@ -511,6 +511,38 @@ class TableauDashboard(StateDashboard, ABC):
     viewPath: str
     filterFunctionName: Optional[str] = None
     filterFunctionValue: Optional[str] = None
+    timezone: str
+    data_tableau_table: str
+    location_name_col: str
+    cmus: Dict[str, CMU]
+
+    def fetch(self) -> pd.DataFrame:
+        return self.get_tableau_view()[self.data_tableau_table]
+
+    def __normalize(self, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
I would stay away from dunder's if possible ([google style guide](https://google.github.io/styleguide/pyguide.html#3162-naming-conventions) gives a good explanation) - would `_normalize` do the trick?